### PR TITLE
fix: GalleryItem Activity table filter interaction

### LINKED ISSUE #7130: GalleryItem Activity table filter interaction
#### Step to reproduce
- go to any `ahk` or `ahp` Item
- click on any button in activity table

```log
message: 'Variable "$interaction" got invalid value "MINTNFT" at "interaction[0]"; Value "MINTNFT" does not exist in "Interaction" enum. Did you mean the enum value "MINT"?'
```

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Bugfix

  ## Needs QA check

  - @kodadot/qa-guild please review

  ## Context

  - [x] Closes #7130 
  - [x] support 'MINT' event on ksm \ ahp \ ahk netwrok

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [x] My fix has changed UI

`/ahk/gallery/26-3`. 

before:
image

after:  
image

  ## Copilot Summary
  
### <samp>🤖 Generated by Copilot at b8920f5</samp>

Improved the `interaction` variable logic and display in the `GalleryItemActivityTable` component. Fixed the GraphQL query to use the correct network value.

  
### <samp>🤖 Generated by Copilot at b8920f5</samp>

> _Sing, O Muse, of the skillful refactorer_
> _Who tamed the `interaction` variable_
> _And made it yield its network-specific name_
> _By using a computed property's power._

### CODE REVIEW

**Comment by prury** [comment]
glad it got fixed! thank you! 

image

File: components/gallery/GalleryItemTabsPanel/GalleryItemActivityTable.vue [hunk 1]
```diff
@@ -126,25 +126,34 @@ onMounted(async () => {
...
-    ? dprops.interactions.filter((i) => i !== 'MINTNFT' && i !== 'CONSUME')
-    : dprops.interactions
+const interaction = computed(() =>
+  dprops.interactions.map((key) => {
+    if (['ksm', 'ahk', 'ahp'].includes(urlPrefix.value)) {
+      switch (key) {
+        case 'MINTNFT':
+          return 'MINT'
+        case 'CONSUME':
+          return 'BURN'
```
Comment (daiagi) [inline components/gallery/GalleryItemTabsPanel/GalleryItemActivityTable.vue:136]: what about default case?

Comment (Jarsen136) [reply components/gallery/GalleryItemTabsPanel/GalleryItemActivityTable.vue:136]: return key by default (check line 139)

**Review by roiLeo — APPROVED** [review]

File: components/gallery/GalleryItemTabsPanel/GalleryItemActivityTable.vue (hunk 1 above)
Comment (daiagi) [reply components/gallery/GalleryItemTabsPanel/GalleryItemActivityTable.vue:136]: sorry mate, i missed that. all good

**Review by daiagi — APPROVED** [review]

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 4.25 USD/DOT ~ 7.059 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x1a232c0ab795ff75da699ebe484733c8c2d3233edae51c7898b4617bc08e3b36](https://polkadot.subscan.io/extrinsic/0x1a232c0ab795ff75da699ebe484733c8c2d3233edae51c7898b4617bc08e3b36)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**